### ISSUE TITLE
improve card ui

### ISSUE THREAD

**Issue by Gaston82** [Gaston82/news-react-app#3]
Change the style for something more minimalistic like a magazine or newspaper.

**Comment by merikettapearl212** [comment]
Hi is this still available? Thank you

**Comment by Gaston82** [comment]
Hey @merikettapearl212 , yes it is. Do you want to work on it?

**Comment by merikettapearl212** [comment]
Yes please, could you also add a hacktoberfest label ? Or does it not need one to count? Thank you

**Comment by Gaston82** [comment]
@merikettapearl212 done!!!

**Comment by merikettapearl212** [comment]
Do you have any ideas in mind, I’m not at my computer so I can’t see anything without the live site link. So idk what it looks like now vs what you would like for it to look like. Thank you

**Comment by Gaston82** [comment]
Hello, there is another issue about home ui where you can see 2 photos with  some ideas.
Thanks @merikettapearl212
